pkg/backends/newrelic/newrelic.go

```diff
@@ -189,7 +191,29 @@ func (n *Client) processMetrics(metrics *gostatsd.MetricMap, cb func(*timeSeries
 
 // SendEvent sends an event to New Relic.
 func (n *Client) SendEvent(ctx context.Context, e *gostatsd.Event) error {
-	return nil
+	select {
+	case <-ctx.Done():
+		return ctx.Err()
+	case buffer := <-n.eventsBufferSem:
+		defer func() {
+			buffer.Reset()
+			n.eventsBufferSem <- buffer
+		}()
+
+		m := make(map[string][]map[string]string)
+		new_event := make(map[string]string)
+
+		/*
+		* NewRelic Events support a summary and a category rather than a
+		* summary and a title. There isn't a clear mapping between these fields
+		* either. Rathern than losing all of the information about an event,
+		* send just the title as the summary of the event.
+		 */
```

Comment:
Do NR events support other fields beyond summary and category?  There's quite a lot of data from the event being lost here.

Reply:
NR Events, can store a max of 255 attributes per event which is essentially a flat JSON object, so there is no reason to drop all the other data. 

Also the comment doesn't make much sense to me with how the NR platform works, and since the gostatsd.Event is pretty flat, all attributes could be retained quite easily. Probably just do some looping for tags to flatten like I've done for the newrelic backend.